### PR TITLE
Fix loading groupfolder info when share api is disabled

### PR DESCRIPTION
**Pull request by CarlSchwan**
Now 
image

### CODE REVIEW

File: apps/dav/lib/Connector/Sabre/File.php [hunk 1]
```diff
@@ -215,16 +215,6 @@ public function put($data) {
 				$data = $tmpData;
 			}
 
-			$data = HashWrapper::wrap($data, 'md5', function ($hash) {
```
Comment (CarlSchwan) [inline apps/dav/lib/Connector/Sabre/File.php:218]: oups should not have been included in this PR

**Review by PVince81 — APPROVED** [review]
:+1:

File: apps/files_sharing/src/views/SharingTab.vue [hunk 2]
```diff
@@ -353,3 +357,10 @@ export default {
 	},
 }
 </script>
+
+<style>
```
Comment (Pytal) [inline apps/files_sharing/src/views/SharingTab.vue:361]:
```suggestion
<style lang="scss" scoped>
```

**Comment by CarlSchwan** [comment]
ci failure unrelated

**Review by Pytal — APPROVED** [review]

**Pull request merged**